rtree/index.py

```diff
@@ -5,24 +5,12 @@
 
 from . import core
 
-try:
-    import cPickle as pickle
-except ImportError:
-    import pickle
+import pickle
 
 import sys
-if sys.version_info[0] == 2:
-    range = xrange
-    string_types = basestring
-elif sys.version_info[0] == 3:
-    string_types = str
 
+string_types = str
```

Comment:
Rather than define this here, on line 223 just replace `string_types` with `str`

Reply:
Makes sense. I've just updated the PR.